okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.kt

```diff
@@ -285,4 +288,23 @@ class HttpLoggingInterceptor @JvmOverloads constructor(
     return !contentEncoding.equals("identity", ignoreCase = true) &&
         !contentEncoding.equals("gzip", ignoreCase = true)
   }
+    private fun mediaTypeIsProbablyBinary(mediaType: MediaType?): Boolean {
+        val type = mediaType?.type
+        val subType = mediaType?.subtype
+
+        val binaryTypes = listOf("audio", "binary", "font", "image", "video")
+        val binaryApplicationSubtypes = listOf("binary", "excel", "java-archive", "java-vm", "lzx",
```

Comment:
I very much do not want to maintain this list of binary media types. Wanna make this the user’s problem by using the same strategy as `headersToRedact`?

I think this also wants us to finally finish this: https://github.com/square/okhttp/issues/782